### DISCUSSION TITLE
Maybe don't recommend repositories when I'm already a contributor

### DISCUSSION THREAD

**Discussion by bokub** [community/community#64488] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why do I see this "recommendation" for a repository? I'm already a contributor, I don't see how this recommendation is relevant

![image](https://github.com/community/community/assets/17952318/d5b1b0d5-d8ed-49f4-9b69-4e9317bb7198)

![image](https://github.com/community/community/assets/17952318/e10bf0f9-791b-4a9e-a958-c56f51c64bac)

**Comment by arafajs** [comment]
The recommendation might be based on your recent activity on GitHub. For example, if you have been viewing or commenting on issues in the repository, GitHub might recommend it to you again.
